docs/Harmony API introduction.ipynb

```diff
@@ -279,10 +279,11 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "According to the services.yml, our `C1233800302-EEDTEST` collection is associated with the `harmony/gdal` service with bounding box and variable subsetting, reprojection, and reformatting. We will request these services below. "
+    "According to the services.yml, our `harmony_example` collection is associated with the `harmony/gdal` service with bounding box and variable subsetting, reprojection, and reformatting. We will request these services below. "
```

Comment:
It's a little confusing to me that we use the short name here when it doesn't appear in the services.yml. Maybe use 1harmony_example/C1256583785-EEDTEST' (or whatever the collection ID is)?

Reply:
This is unrelated to the no-op collection. This is the `harmony-example` collection that was used throughout the notebook. It is referred as the `harmony-example` collection throughout the notebook rather than `C1234088182-EEDTEST`.